### ISSUE TITLE
Issue deleted

### ISSUE THREAD

**Issue by boracasli7** [Tatoeba/tatoeba2#1076]
I know here isn't the right place to post this, but I had 

to post here.

1. Please don't delete this issue, even if you will close 

this. I will only contribute sentences in my strongest 

languages, which happen to be Turkish (native), English 

(advanced), Greek (advanced), Georgian (intermediate) and 

Swedish (intermediate).

2. I have Asperger's, that means I'm an addict of 

something and I have social problems. By the way, I would 

never harm anyone and anything either in real or virtual 

life, and I'll also try not to get angry. My anger cost my 

previous computer and I'm still feeling sad for doing 

that.

3. I never contributed sentences in the languages I 

couldn't speak even a single word, you should know this, 

but I contributed sentences in my weaker languages in the 

past. I won't do that again.

4. Tatoeba is not 100% accurate and even natives can make 

mistakes when writing in their native languages. So please 

don't expect Tatoeba to be absolutely perfect.

5. I apologise, it was my bad, because I talked badly 

about you and others and you were right. I didn't mean to 

insult you (or anyone), but my aggressivity made me do 

that, because I sometimes get angry very easily. Because 

of this weakness of mine, I hate myself. But I promise, 

I'll try to control my behaviours.

6. I know that I'm not the only one there, but I see some 

people contributing in many languages or in languages 

which they don't even speak a single word or using machine 

translation when translating and some of them have never 

been warned for anything. I'll stop adding sentences in my 

weaker languages, and I'd be very glad if you welcomed me 

there again.

7. We have to stop the hostility against each other. 

Tatoeba is not a social network, but anyways, it helps us 

make more friends. Let's become friends again!

8. I promise I'll obey what I've just written to you.

Thank you very much for reading this.

Best wishes,
Bora

**Comment by RyckRichards** [comment]
Nobody was hostile with you, but you were hostile to everybody.

**Comment by RyckRichards** [comment]
Issue reopened and waiting for @trang 's pronunciation.

**Comment by trang** [comment]
@boracasli7, I would like you to close this issue yourself, as a confirmation that you understand the following:

1. Please stop creating issues in our GitHub. I have already told you in the past, and I will tell you again: you must contact us by email at community-admins@tatoeba.org. Creating issues in GitHub will not help you to be accepted again in Tatoeba.

2. For now, the best offer I can give you is that you use our alternate website: https://dev.tatoeba.org. On that website you can use your account [boracasli](http://dev.tatoeba.org/eng/user/profile/boracasli). I have unbanned it, only on dev.tatoeba.org. We won't ban you over there, I can guarantee it.

3. There are several features that we need to implement before we can accept you again in tatoeba.org. It will take time to implement these features, therefore you will have to be patient. I cannot give you an exact estimation of when we can implement these features, it could be 1 year, or 2 years, or 3 years. But I hope that using the alternate website (https://dev.tatoeba.org) will be enough in the meantime, to appease your addiction.

If you understand these 3 points, then please close this issue yourself. Thank you.

**Comment by boracasli7** [comment]
Then I'll leave your fucking site!

**Comment by boracasli7** [comment]
Please ban all my accounts on the dev site too! (Keskin, LanguageIsLove, boracasli)

**Comment by boracasli7** [comment]
Please

**Comment by boracasli7** [comment]
Your boobies are fucked

**Comment by boracasli7** [comment]
fucklhfjrufewueduopewgfopuudsfopu

**Comment by boracasli7** [comment]
wklefklklgrjklgkl

**Comment by boracasli7** [comment]
Frenches are stupid as fuck

**Comment by boracasli7** [comment]
Fuck france

**Comment by boracasli7** [comment]
then i'll flood and hack into your twitter and facebook

**Comment by boracasli7** [comment]
fuck off

**Comment by boracasli7** [comment]
The dev site is shit

**Comment by boracasli7** [comment]
Reblock me again on dev... Revoke access to dev site

**Comment by boracasli7** [comment]
Dev site is gonna be banned

**Comment by boracasli7** [comment]
Block my accounts also on the dev site

**Comment by boracasli7** [comment]
And I'm not a dev. Devs are shitty people and you should know it

**Comment by boracasli7** [comment]
Your boobs are gonna milk

**Comment by boracasli7** [comment]
I'm gonna milk your boobs
